
This patch will not be send upstream because pkg-config (.pc) files
are not able to suppress of emit of -L -R -I options for standard
paths by an efficient way also for MACH64 .

--- a/libcurl.pc.in	2017-10-22 22:51:20.000000000 +0000
+++ b/libcurl.pc.in	2020-06-24 19:13:05.788779943 +0000
@@ -34,6 +34,6 @@
 URL: https://curl.haxx.se/
 Description: Library to transfer files with ftp, http, etc.
 Version: @CURLVERSION@
-Libs: -L${libdir} -lcurl @LIBCURL_NO_SHARED@
+Libs: -lcurl @LIBCURL_NO_SHARED@
 Libs.private: @LIBCURL_LIBS@
-Cflags: -I${includedir} @CPPFLAG_CURL_STATICLIB@
+Cflags: @CPPFLAG_CURL_STATICLIB@
